priv/repo/seeds.exs

```diff
@@ -237,18 +219,32 @@ units =
 
 Repo.insert_all(Unit, units, on_conflict: :nothing)
 
+# Add each level's rewards
 currency_rewards =
-  Enum.map(Enum.with_index(levels_without_units, 1), fn {level, level_index} ->
+  Enum.map(levels_without_units, fn level ->
     %{
       level_id: level.id,
-      amount: 10 * level_index,
-      currency_id: gold.id,
+      amount: 10 * (20 + level.level_number),
+      currency_id: gold_currency.id,
       afk_reward: false,
       inserted_at: NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second),
       updated_at: NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second)
     }
   end)
 
+currency_rewards =
+  currency_rewards ++
+    Enum.map(levels_without_units, fn level ->
+      %{
+        level_id: level.id,
+        amount: (10 * (15 + level.level_number - 1) * 1.025) |> round(),
```

Comment:
We could create a private function for the amounts calculation for each currency and mention there the spreadsheet. Otherwise this calculation looks random.

Reply:
As the seeds file is not a module, I added a comment explaining the origin of this calculation, referencing the spreadsheet. 
If the need of delegating more behavior to private functions increases in the future we should have a utils module.